packages/swap/src/translators/reactjs/state/state.ts

```diff
@@ -115,27 +119,27 @@ export const defaultSwapState: SwapState = {
     datumHash: '',
     amounts: {
       sell: {
-        quantity: '0',
+        quantity: Quantities.zero,
         tokenId: '',
       },
       buy: {
-        quantity: '0',
+        quantity: Quantities.zero,
         tokenId: '',
       },
     },
     slippage: 1,
-    limitPrice: undefined,
+    limitPrice: Quantities.zero,
```

Comment:
shouldn't be undefined by default?

Reply:
As soon as a pool is selected it's gonna be set to the market price. Until then, an undefined needs to be handled on every function that takes the limit price, while a zero works.